content/api/commands/Maximizing Sessions in Cypress.md

```diff
@@ -0,0 +1,325 @@
+# Maximizing Sessions in Cypress
+
+As you likely know, Cypress ensures consistent results through test isolation,
+where test isolation is the practice of resetting the state _before_ each test.
+
+Cleaning up state ensures the operations of one test does not affect another
+test downstream. When writing tests, the goal for the test is reliably pass when
+ran standalone or in a randomized order. If you have set up your tests in a way
+that requires the state of a previous test, this could potentially break your
+tests.
```

Comment:
```suggestion
Cleaning up state ensures the operation of one test does not affect another
tests downstream. When writing tests, the goal for the test is reliably pass when
run standalone or in randomized order. If you have set up your tests in a way
that requires the state of a previous test, this could potentially break your
tests.
```